site/frontend/src/pages/compare/shared.ts

```diff
@@ -78,3 +78,18 @@ export function getBoolOrDefault(
   }
   return defaultValue;
 }
+
+export function isBenchmarkNotFilteredByName(
```

Comment:
This name is a bit.. unintuitive :laughing: What about something like `benchmarkNameMatchesFilter`?